app/src/main/res/values/strings.xml

```diff
@@ -346,7 +346,7 @@
         <item>Three friends are playing a game involving black and white hats. At the beginning of the game, each person is randomly assigned a hat color, black or white, independently from everyone else and with 50-50 probability.
         \n\nImagine that people cannot see what\'s on their own head, and only observe the colors of their two friends\' hats. No communication is allowed. The players must simultaneously guess the color of their own hat (or decline to guess). The entire group wins a prize if at least one of them guesses correctly and nobody guesses incorrectly.
         \n\nSuppose the team adopts a \"contrarian\" strategy: anyone who observes two white hats will guess that their own hat is black; anyone who observes two black hats will guess that their own hat is white; and anyone who observes anything else will decline to guess. At the beginning of the game (before the hat colors are assigned), what is the probability that the group wins the prize?</item>
-        <item>How many different triangles can you form from the seven lines below? Note that none of the lines are parallel, and imagine that they extend infinitely far beyond the edges of the image.</item>
+        <item>How many different triangles can you form from the seven lines below? Note that none of the lines are parallel, and imagine that they extend infinitely far beyond the edges of the image. We assume that no more than 2 lignes crosses at the same point.</item>
```

Comment:
Good catch, and thank you for opening this pull request!

Do you mind if I add a commit on this branch changing "2 lignes crosses" to "two lines cross"?

I should also create a new version of the puzzle image in which those three lines very clearly do not cross at the same point.

Reply:
Sure... My English isn't perfect =)

Reply:
All good!  Merci de m'avoir aidé à améliorer l'appli.